src/main/Extensions/VSCode/VSCodeExtension.ts

```diff
@@ -172,7 +172,7 @@ export class VSCodeExtension implements Extension {
             defaultAction: {
                 handlerId: "Commandline",
                 description: `Open ${fileType} in VSCode`,
-                argument: template.replace("%s", `${commandArg} ${uri}`),
+                argument: template.replace(/"?%s"?/, `${commandArg} ${uri}`),
```

Comment:
This means optional " but thats already the case just remove the quotes from the template in the settings. Also send what template you are using

Reply:
![image](https://github.com/user-attachments/assets/21f5c845-86b5-455c-b2a9-7e808de46a57)
just remove the quotes from the template to achieve the same thing